var/spack/repos/builtin/packages/rocm-tensile/package.py

```diff
@@ -18,7 +18,8 @@ class RocmTensile(CMakePackage):
 
     license("MIT")
 
-    maintainers("srekolam", "renjithravindrankannath", "haampie")
+    maintainers("srekolam", "renjithravindrankannath", "haampie", "afzpatel")
+    version("6.3.0", sha256="553482df07196fd90e3a02a12dcfe798a68f2113618b178086b63384d8982682")
```

Comment:
I didn't get this sha256; rather:

```suggestion
    version("6.3.0", sha256="7ae90d1a513dc6f000a45f644b360305ef212ab3dff7b0217b6addabebf932e1")
```

Is that expected?

Reply:
thanks for catching that